presto-docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -511,6 +511,25 @@ dropping the table from the metadata catalog using ``TRUNCATE TABLE``.
     -----------+------+-----------+---------
     (0 rows)
 
+DELETE
+^^^^^^^^
+
+The iceberg connector can delete data in one or more entire partitions from tables by using ``DELETE FROM TABLE``::
+
+     DELETE FROM lineitem;
+
+     DELETE FROM lineitem WHERE linenumber = 1;
+
+     DELETE FROM lineitem WHERE linenumber not in (1, 3, 5, 7) and linestatus in ('O', 'F');
+
+.. note::
+
+    Columns in the filter must all be identity transformed partition columns of the target table.
+
+    Do not support non-comparison operator for the filter columns, for example ADD, SUBTRACT, MODULUS, etc.
```

Comment:
Latin such as etc., e.g, i.e, should be avoided. For an example of a standard for this, see the [GitLab recommended word list entry for etc.](https://docs.gitlab.com/ee/development/documentation/styleguide/word_list.html#etc).  To address this adapting @ZacBlanco's suggestion, consider something like this. 

```suggestion
    Filtered columns only support comparison operators, such as EQUALS, LESS THAN, or LESS THAN EQUALS.
```